companies/zucchetti.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "zucchetti",
+    "relevant-countries": [
+        "it",
+        "us",
+        "spa",
+        "br",
+        "fr"
+    ],
+    "categories": [
+        "it",
+        "commerce"
+    ],
+    "name": "Zucchetti",
+    "address": "Via Solferino 1, 26900 Lodi - Italia",
+    "phone": "+39 0371 594-1",
```

Comment:
Please make sure the number is in the correct international format. You can either use the company JSON generator for that or use [libphonenumber](https://libphonenumber.appspot.com/).